docs/components/skip_link.md

```diff
@@ -0,0 +1,23 @@
+
+# Skip Link
+
+```django
+{% include 'wagtailnhsukfrontend/skip_link.html' %}
+```
+| Option | Description | Default |
+| ------ | ----------- | ------- |
+| `text` | Title to display for the link | `Skip to main content` |
+| `href` |  The href tag for the skip link| `#maincontent` |
+
+To ensure that the skip link works as intended make sure its the first include in the template. You can also pass the two options above using `with` tag after the `include` tag.
```

Comment:
Technically `with` isn't a tag.

You can probably remove this sentence by just including an example instead:
`{% include 'wagtailnhsukfrontend/skip_link.html' with text="Skip to article" href="#article" %}`